docs/usage-and-billing.mdx

```diff
@@ -0,0 +1,158 @@
+---
+slug: usage-and-billing
+append_help_link: true
+title: Usage and billing 
+description: "Learn about pricing, tiers, and feature support for the following Semgrep Products: Semgrep OSS, Semgrep Code, and Semgrep Supply Chain."
+tags:
+  - Semgrep Cloud Platform
+  - Semgrep OSS
+  - Team & Enterprise Tier
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp";
+
+<ul id="tag__badge-list">
+  {Object.entries(frontMatter)
+    .filter((frontmatter) => frontmatter[0] === "tags")[0]
+    .pop()
+    .map((value) => (
+      <li class="tag__badge-item">{value}</li>
+    ))}
+</ul>
+
+<!-- # Pricing and billing -->
+
+# Usage and billing 
+
+Learn about usage computation and other aspects of your Semgrep licenses.
+
+:::note
+- This document is for **Semgrep Pro** users; Semgrep OSS Engine does not have any usage limit. Compare [<i class="fa-regular fa-file-lines"></i>  Semgrep Pro and Semgrep OSS](/semgrep-pro-vs-oss).
+:::
+
+## Usage 
+
+All Semgrep Pro products are free under the **Team** tier for **10 monthly contributors**. These products include:
+
+- Semgrep Cloud Platform
+- Semgrep Code
+- Semgrep Supply Chain
+- Semgrep Secrets (beta)
+
+A **contributor** is someone who has made at least **one** commit to a Semgrep-scanned private repository within the last month, starting from the **date of license purchase** if a license was purchased, or the date of account creation, for accounts using Semgrep within usage limits.
+
+Any type of Semgrep Pro scan counts towards the contributor total. This includes:
+
+- Scanning with any Semgrep Pro product (Code, Supply Chain, and so on.)
+- Full scans on a repository or partial scans on a pull or merge request.
+
+### Usage computation
+
+Contributors are calculated using `git log` over the **past 30 days** (a rolling interval), not the beginning of each month. The start date is either:
+
+- The date of your license purchase.
+- The date of account creation, if you and your team are within usage limits.
+
+**Bots** and similar automations are excluded from this count.
+
+### Exceeding the free usage limit
+
+Semgrep scans stop when the usage limit is exceeded. You can resume scanning through the following:
+
+* A one-time 30-day free trial that starts automatically when the usage limit is exceeded for the first time.
+* By purchasing additional licenses.
+* By waiting for the next billing cycle.
```

Comment:
The through/by combo here is kind of awkward. Maybe it would work with 

...resume scanning through:

* A one-time...
* Purchasing...
* Waiting...

?